Docs/quickstart.md

```diff
@@ -226,6 +227,14 @@ The `"DecoderValueSensor"` decoder is not a best practice but it makes it easier
 
 if the SensorDecoder tag has a "http" in it's string value, it will forward the decoding call to an external decoder, as described in option 2 above, using standard Http. The call expects a return value with the same format as the json here above or an error string.
 
+There is an unsupported [repository](https://github.com/TechPreacher/iotedge-lorawan-starterkit-decoders) containing sample implementations for the following, commercially available LoRaWAN sensors, based on the DecoderSample contained in this repo:
```

Comment:
As discussed we should not have it in here

Reply:
Removed!